docs/audits/2024_q4/bdk_audit_report.md

```diff
@@ -0,0 +1,1031 @@
+---
+title: "Wizardsardine BDK Audit Report"
+description: "2024 Code Audit by Wizardsardine"
+authors:
+ - Antoine Poinsot 
+date: "2024-10-21"
+tags: ["BDK", "project"]
+draft: true
+---
+
+[Spiral](https://spiral.xyz/) hired [Wizardsardine](https://wizardsardine.com/) for 10 days to perform
+a (partial) audit of the [BDK wallet library](https://github.com/bitcoindevkit/bdk) (at tag
+`v1.0.0-beta.2`).
+
+This audit was focused on the `bdk_chain`, `bdk_electrum`/`bdk_esplora` and `bdk_wallet` crates with
+the intention to look for defects which could lead to returning incorrect information to a user
+about their wallet or to prevent them from reliably using it under realistic circumstances (e.g. a
+crash bug which can be triggered by a third party, or a significant slowdown in performing some
+operations).
+
+A threat model i considered interesting for the purpose of this review was that of a server-side
```

Comment:
Shouldn't `I` always be capitalized?

Reply:
I was planning to leave the report as-is, but you're right that we should fix simple things like this.